addons/api/mirage/generated/factories/managed-group.js

```diff
@@ -0,0 +1,25 @@
+import { Factory } from 'ember-cli-mirage';
+import { random, date, datatype } from 'faker';
+
+/**
+ * GeneratedManagedGroupFactory
+ * A managed group is a resource that represents a collection of accounts.
```

Comment:
It represents a collection of users I thought?

Reply:
I am happy to change it but I grab it [from here](https://www.boundaryproject.io/docs/concepts/domain-model/managed-groups).

Reply:
Ah you are correct.  This indicates that `member_ids` is an array of accounts, not users.

Reply:
But double check with BE.